tests/ai_unet/test_model_training.py

```
@@ -0,0 +1,364 @@
+        TEST NAME: test_dice_scores_calculation
+        COMPONENT: dice_scores
+        1. Create dummy logits and target tensors.
+        2. Call dice_scores().
+        3. Assert the returned tuple matches expected dice values.
+        """
+        logits = torch.tensor([[[[1.0, 1.0], [-1.0, -1.0]], 
+                                [[1.0, -1.0], [-1.0, -1.0]]]])
+        targets = torch.tensor([[[[1.0, 1.0], [0.0, 0.0]], 
+                                 [[1.0, 1.0], [0.0, 0.0]]]])
+        
+        d_score, c_score, mean_score = model_training.dice_scores(logits, targets)
```

Ruff (0.15.12)

[warning] Unpacked variable `mean_score` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)